deps/spidershim/test/v8.cc

```diff
@@ -11,5 +11,5 @@
 #include "gtest/gtest.h"
 
 TEST(SpiderShim, Version) {
-  v8::V8::GetVersion();
+  EXPECT_EQ(strncmp("JavaScript-C", v8::V8::GetVersion(), 12), 0);
```

Comment:
The order of arguments is the reverse (expected first).  I'll fix after merging.